src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

Reply:
Why we need to do this? Can the return type be a Enum type? Class Ball has field Color of Enum type.